translations/translation_hu.xml

```diff
@@ -502,63 +501,63 @@ Ezt követően mindig változtathatsz más értékekkel és generálhatsz útvon
+Központi opciók:
+- Mező közepe: Különböző módok vannak arra, hogy miként nézzen ki a mező középső mintázata. A klassikus és legtöbbet használt opció az fel/le.
+Spirális, Versenypálya és Blokk mind saját előnyökkel rendelkezik a többi módhoz képest. Blokk opció esetén például a kombájn csöve legtöbbször a terményből kifelé mutat a könnyebb kirakodásért.
+- Fel/le irány: Általában az automatikus opció megtalálja a legjobb irányt, de néha a leghosszabb él jobban passzol. Ha egyik sem felel meg neked, válaszd a kézi módot és állítsd be az irányt magad.
+- Kihagyandó sorok: Ez egy nagyon hasznos lehetőség a munkád felgyorsításához, mivel a gépeknek nem kell tolatniuk a következő sorra való forduláshoz.
+- Sorok blokkonként: Ennek csak akkor van hatása, ha a mező közepe blokkokar van állítva és megmondja a generátornak, hogy hány sor legyen egy blokkban. Ha több a sor, kevesebb blokk lesz generálva.
+- Fel/le sorok szöge: Amikor a fel/le irány mellett a kézi módot választod, ez a beállítás megmondja a generátornak a sorok irányszögét. 0° jelenti az Észak-Dél irányt.
 "/>
     <text name="CP_help_page_courseGeneratorHeadland_text" text="
-Headland settings will only show up, when you have set at least 1 at Number of headlands.
-You gain options for where to start on, corner settings, direction and overlap.
-Headlands are highly suggested to prevent tools leaving the field when turning.
+A fordulósáv beállítások csak akkor jelennek meg, ha a fordulósávok számát legalább 1-re állítod.
+Ezzel olyan lehetőségeket nyersz, mint a kezdés helye, sarok beállítások, irány és átfedés.
+A fordulósávok használata erősen ajánlott a fordulás közbeni mező elhagyás megelőzése érdekében.
 "/>
     <text name="CP_help_page_courseGeneratorHeadlandFunctions_text" text="
-- Start work on: As mentioned in the extended AI Menu, the field position is used to set the start or end position for the field work course.
-When set to start on headland, it tells where the start should be. When changed to start on rows, it will tell where to end the work.
-- Headland corners: Smooth will have a simple smoothing over the corners, This works for all tools, but will lead to tiny unworked patches in the corners.
-They can be reduced with overlap, but never disappear. Sharp will generate a turn based on the angle of the field border angle.
-Round will take the current selected vehicles turn radius into account to have the best round corner for that tool.
-- Headland direction: clockwise or counter-clockwise. This can be important for combines and harvesters, depending on wich side the pipe or conveyor belt is.
-- Headland overlap: How much % overlap all headlands should have. The first one will have no overlap with the field border, but the inner ones will overlap with their neighbour.
+- Munka kezdés helye: Ahogy említésre kerül a bővített AI menüben, a mező pozíciója arra szolgál, hogy a mezőmunka utvonal kezdetét vagy végét megadjuk.
```

Comment:
Ahogy a bővitett AI menüben is láttuk vagy valami ilyesmi inkább, a kerül az nekem nagyon hivatalos.